doc/source/index.rst.template

```diff
@@ -21,7 +23,85 @@ pandas: powerful Python data analysis toolkit
 easy-to-use data structures and data analysis tools for the `Python <https://www.python.org/>`__
 programming language.
 
-See the :ref:`overview` for more detail about what's in the library.
+.. raw:: html
+
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 d-flex">
+                <div class="card text-center intro-card shadow">
+                <img src="_static/index_getting_started.svg" class="card-img-top" alt="getting started with pandas action icon" height="52">
+                <div class="card-body flex-fill">
+                    <h5 class="card-title">Getting started</h5>
```

Comment:
Can you make each of these titles hyperlinks to the right section?

Reply:
> Can you make each of these titles hyperlinks to the right section?

I can, but the problem is that I can only do this with hardcoded html links (so not with a sphinx label that will always work, even if we rename a page that it is referencing). 

That's the reason I mixed some sphinx rst in between the raw html for the buttons, to still get the link through sphinx.  
Now, for those 4 main links, that's maybe not a big problem.

(I think ideally, we would write a custom sphinx directive for those blocks, so we can write in normal text and let the directive generate the hmtl, instead of using raw html here, but that would require some more work)